pages/githubandmarkdown.md

```diff
@@ -67,18 +67,17 @@ Please, make sure to ALWAYS work on your own personal account via forking and pu
 
 
 ###Create a new file as your personal MD Wiki page and commit your changes
-Now, you can edit the files in your repo and the changes you make will be reflected on your &lt;username&gt;.github.io website. To edit, click on the `pages` folder. Then, click `New File` and name the file exactly the same as your GitHub username. We created pages for you using your GitHub username, so it is important that you stick with that name. For reference, an example of a link to a personal page is EmilyLarkin.github.io/pages/EmilyLarkin.md, so yours should be named in the same manner, only using your username.
+ Before creating your file create a new branch using the branch drop down menu as shown below.
+ 
+![New Branch](https://cloud.githubusercontent.com/assets/5189447/24425120/f08696a2-13d0-11e7-8644-d9df3142e139.png)
```

Comment:
You need to crop the image to the relevant parts (people don't need to see my browser tabs or task bar for **example**)

Also you need to add the image to the repo - we don't want the github cloud link.